ci-scripts/osx/travis-build.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
+pushd toonz/sources
+chmod +x beautification.sh
```

Comment:
thx. I fixed it on windows by `git update-index --add --chmod=+x toonz/sources/beautification.sh`.